main.js

```diff
@@ -0,0 +1,430 @@
+$(document).ready(initializeGame);
+
+function initializeGame(){
+    attachClickHandlers();
+    currentPlayer = player[0];
+    $(".player1").find(".token1").hide();
+};
+function attachClickHandlers(){
+    $(".forfeitButton").on("click", resetGame);
+    $(".column").on("click", piecePlacement);
+
+};
+
+
+// GLOBAL VARIABLES
+
+var column_clicked = null;
+var row_clicked = null;
+var rowToPlacePiece = null;
+var putPieceInRow = 0;
+var games_played = 0;
+var attempts = 0;
+var piecesInColumn = 0;
+// ----------------
+
+
+var player = [];
+player[0] = {
+    name: "player1",
+    points: 0,
+    gamesWon: 0,
+    gamesLost: 0,
+    gamesTied: 0,
+};
+player[1] = {
+    name: "player2",
+    points: 0,
+    gamesWon: 0,
+    gamesLost: 0,
+    gamesTied: 0,
+};
+
+
+function placePlayerToken(piecesInRow) {
+    if (currentPlayer === player[0]) {
+        gameArray[piecesInRow][column_clicked].token = 1;
```

Comment:
using global variables like this can lead to race conditions.  You are better off passing them into the function

Reply:
Also you could have put in the player objects which number they were.

```
player[1] = {
   name: "player2",
   tokenID: 1 ...
```
and then here

```
gameArray[piecesInRow[column_clicked.token = currentPlayer.tokenID;
```